include/ck/tensor_operation/gpu/device/device_grouped_gemm_bias_transpose_xdl.hpp

```diff
@@ -0,0 +1,711 @@
+                          arg.gemm_desc_kernel_arg_.data(),
+                          arg.gemm_desc_kernel_arg_.size() * sizeof(GemmBiasTransKernelArg),
+                          hipMemcpyHostToDevice));
+
+            float ave_time = 0;
+
+            if(has_main_k_block_loop)
+            {
+                const auto kernel =
+                    kernel_grouped_gemm_bias_transpose_xdlops_v2r3<GridwiseGemm,
+                                                                   GemmBiasTransKernelArg,
+                                                                   AElementwiseOperation,
+                                                                   BElementwiseOperation,
+                                                                   CElementwiseOperation,
+                                                                   true>;
+
+                ave_time = launch_and_time_kernel(
+                    stream_config,
+                    kernel,
+                    dim3(arg.grid_size_),
+                    dim3(BlockSize),
+                    0,
+                    cast_pointer_to_constant_address_space(arg.gemm_descs_args_workspace_),
+                    arg.gemm_desc_kernel_arg_.size(),
+                    arg.a_element_op_,
+                    arg.b_element_op_,
+                    arg.c_element_op_);
+            }
+            else
+            {
+                const auto kernel =
+                    kernel_grouped_gemm_bias_transpose_xdlops_v2r3<GridwiseGemm,
+                                                                   GemmBiasTransKernelArg,
+                                                                   AElementwiseOperation,
+                                                                   BElementwiseOperation,
+                                                                   CElementwiseOperation,
+                                                                   false>;
+
+                ave_time = launch_and_time_kernel(
+                    stream_config,
+                    kernel,
+                    dim3(arg.grid_size_),
+                    dim3(BlockSize),
+                    0,
+                    cast_pointer_to_constant_address_space(arg.gemm_descs_args_workspace_),
+                    arg.gemm_desc_kernel_arg_.size(),
+                    arg.a_element_op_,
+                    arg.b_element_op_,
+                    arg.c_element_op_);
+            }
+
+            return ave_time;
+        }
+
+        // polymorphic
+        float Run(const BaseArgument* p_arg,
+                  const StreamConfig& stream_config = StreamConfig{}) override
+        {
+            return Run(*dynamic_cast<const Argument*>(p_arg), stream_config);
+        }
+    };
+
+    static constexpr bool IsValidCompilationParameter()
+    {
+        // TODO: properly implement this check
+        return true;
+    }
+
+    static bool IsSupportedArgument(const Argument& arg)
```

Comment:
always support all the GEMM sizes?

Reply:
Already filter out unsupported Gemm with CheckValidity in constructor